helpers/grid.css

```diff
@@ -0,0 +1,3 @@
+.grid {
```

Comment:
Don't make different css files for these utility classes just put them in one file !!

Reply:
Implemented [link](https://github.com/tushartiwari7/elevate-ui/pull/2/commits/a73e93dd139013d4aa7485a63845f54da4e5da61)